smpe/bld/SMPMCS.txt

```diff
@@ -24,20 +24,19 @@
 ++SAMP(ZWE6DDEF)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(1) .
 ++SAMP(ZWE7APLY)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(1) .
 ++SAMP(ZWE8ACPT)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(1) .
+++SAMP(ZWEMCOPY)     SYSLIB(SZWEEXEC) DISTLIB(AZWESAMP) RELFILE(2) .
+++SAMP(ZWEOCOPY)     SYSLIB(SZWEEXEC) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWECSVSM)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWEKRING)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWENOKYR)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWENOSEC)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
-++SAMP(ZWENOSSO)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWESECUR)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
-++SAMP(ZWESSOTK)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWESIPRG)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWESIP00)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWESISCH)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWESASTC)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWESISTC)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWESLSTC)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
-++SAMP(ZWESVSTC)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
```

Comment:
just a comment to make you aware that you are removing a STC here. It could be intentional, but is not related to SMPE changes, so I figured I better signal it.